lib/api/src/modules/stories.ts

```diff
@@ -26,7 +27,7 @@ import {
   SetStoriesPayloadV2,
 } from '../lib/stories';
 
-import { Args, ModuleFn } from '../index';
+import { Args, ModuleFn, useAddonState } from '../index';
```

Comment:
Unnecessary import?

Reply:
Where did that come from? Must have been VS code being too smart for its own good.